python/langsmith/schemas.py

```diff
@@ -63,7 +65,7 @@ def my_function(bar: int, my_val: Attachment):
     data: bytes
 
 
-Attachments = Dict[str, Union[Tuple[str, bytes], Attachment]]
+Attachments = Dict[str, Union[Tuple[str, bytes], Attachment, Tuple[str, Path]]]
```

Comment:
It's probably worth making `data` within `Attachment` `data: Union[bytes, Path]`.